jujugui/static/gui/src/app/components/panel/_panel.scss

```diff
@@ -25,4 +25,5 @@
     background-color: rgba(255, 255, 255, 0.3);
     border-radius: none;
     box-shadow: none;
+    overflow: scroll;
```

Comment:
This will break our other panels. Could you try adding the scroll to a child of the search results panel?